frontend/packages/dev-console/integration-tests/cypress.json

```diff
@@ -0,0 +1,25 @@
+{
+    "baseUrl": "http://localhost:9000",
+    "integrationFolder": "features",
+    "testFiles": "**/*.{feature,features}",
+    "defaultCommandTimeout": 30000,
+    "viewportWidth":1920,
+    "viewportHeight":1080,
+    "watchForFileChanges": true,
+    "chromeWebSecurity": true,
+    "waitForAnimation": true,
+    "animationDistanceThreshold": 20,
+    "execTimeout": 90000,
+    "pageLoadTimeout": 90000,
+    "requestTimeout": 15000,
+    "responseTimeout": 15000,
+    "supportFile": "support/commands/index.ts",
+    "pluginsFile": "plugins/index.js",
+    "fixturesFolder": "testData",
+    "video": true,
+    "screenshotsFolder": "screenshots",
```

Comment:
Will dev-console screenshots & videos end up in `frontend/gui_test_screenshots/cypress/...`?  This is important for `./test-prow-e2e.sh` which during CI copies files from `frontend/gui_test_screenshots/...` to [Artifacts](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_console/6475/pull-ci-openshift-console-master-e2e-gcp-console/1308433615603896320/artifacts/e2e-gcp-console/gui_test_screenshots/) folder for PRs.

`frontend/packages/integration-tests-cypress/cypress.json`:
```
 "screenshotsFolder": "../../gui_test_screenshots/cypress/screenshots",
  "videosFolder": "../../gui_test_screenshots/cypress/videos",
```

Reply:
updated cypress.json folder

Reply:
lgtm, thanks!